nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -2414,6 +2413,10 @@ public ErrorBuilder getErrorBuilder() {
     return errorBuilder;
   }
 
+  public Config getConfig() {
```

Comment:
Sorry if this is unbelievably pedantic but: public method -> javadoc 😅 (Also, as a getter, maybe worth moving further up within this file? Closer to the field unless that breaks GJF rules?)